mino/minogrpc/certs/example_test.go

```diff
@@ -0,0 +1,40 @@
+package certs_test
+
+import (
+	"fmt"
+
+	"go.dedis.ch/dela/mino/minogrpc"
+	"go.dedis.ch/dela/mino/minogrpc/certs"
+	"go.dedis.ch/dela/mino/minogrpc/session"
+	"go.dedis.ch/dela/mino/router/tree"
+)
+
+func ExampleStorage_Fetch() {
+	router := tree.NewRouter(minogrpc.NewAddressFactory())
+
+	m, err := minogrpc.NewMinogrpc(minogrpc.ParseAddress("127.0.0.1", 0), router)
+	if err != nil {
+		panic("couldn't start mino: " + err.Error())
+	}
+
+	store := certs.NewInMemoryStore()
+
+	digest, err := store.Hash(m.GetCertificate())
+	if err != nil {
+		panic("certificate digest failed: " + err.Error())
+	}
+
+	err = store.Fetch(m.GetAddress().(session.Address), digest)
```

Comment:
Naive question, I haven't looked into this too much: 
is the cast expected and needed, or is there a better alternative through the API ?

Reply:
When used internally by minogrpc, it knowns the address as session.Address, but for the example I have to get the address from the instance which returns the interface. I could create the address from the host value though.